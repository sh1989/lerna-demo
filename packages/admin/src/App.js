import { Notification } from 'common';
import React from 'react';
import Title from './Title';

const App = () => (
    <div>
        <h1>Admin App!</h1>
        <Title />
        <Notification msg="Shh! This is the secret app!" />
    </div>
);

export default App;