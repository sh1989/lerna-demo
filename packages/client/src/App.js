import { TextBox } from 'common';
import React from 'react';

const App = () => (
    <div>
        <h1>Client App</h1>
        <TextBox>
            <p>Hi!</p>
        </TextBox>
  </div>
);

export default App;