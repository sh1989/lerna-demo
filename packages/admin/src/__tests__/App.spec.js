import React from 'react';
import App from '../App';
import ShallowRenderer from 'react-test-renderer/shallow';

test('App Component', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
        <App />
    );
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
});