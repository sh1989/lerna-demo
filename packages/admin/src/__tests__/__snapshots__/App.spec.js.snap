// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component 1`] = `
<div>
  <h1>
    Admin App!
  </h1>
  <Connect(Title) />
  <Notification
    msg="Shh! This is the secret app!"
  />
</div>
`;
